public/scripts/services/form.js

```diff
@@ -25,6 +25,26 @@
                         value = null;
                     }
                     break;
+                case 'string-datetime':
+                    value = value.toString();
+                    if (value.length === 0) {
+                        value = null;
+                    } else {
+                        function prefixWithZero(value, minLength = 2) {
+                            value = value + "";
+                        
+                            while (value.length < minLength) {
+                                value = `0${value}`;
+                            }
+                        
+                            return value;
+                        }
+                        
+                        const date = new Date(value);
+                        const isoTime = `${date.getUTCFullYear()}-${prefixWithZero(date.getUTCMonth() + 1)}-${prefixWithZero(date.getUTCDate())}T${prefixWithZero(date.getUTCHours())}:${prefixWithZero(date.getUTCMinutes())}:${prefixWithZero(date.getUTCSeconds())}.${prefixWithZero(date.getUTCMilliseconds(), 3)}+00:00`;
```

Comment:
Can we use [`Date.toISOString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) instead?

Reply:
What toISOString gives is `2022-09-11T13:00:00.000Z`, while our standard is ISO 8601 that looks like `2022-09-11T13:00:00.000+00:00`.

It.. Seems to work with both? Not sure why, might be some PHP magic that we shouldn't relay on. Doing it this hectic way ensures we use exact format the format of datetime that we have many tests for, I would keep it.